dashboard/app/controllers/curriculum_tracking_pixel_controller.rb

```diff
@@ -0,0 +1,22 @@
+require 'cdo/firehose'
+
+class CurriculumTrackingPixelController < ApplicationController
+  def index
+    prevent_caching
+    curriculum_page = URI.unescape(params[:from])
+    user_id = current_user&.id
+
+    FirehoseClient.instance.put_record(
+      study: 'curriculum-builder-page-views',
+      study_group: 'v0',
+      event: 'curriculum-builder-page-view',
+      data_json: {
+        curriculumBuilderUrl: curriculum_page,
+        unplugged: curriculum_page.include?("unplugged"),
```

Comment:
Removing `unplugged` since I explored CurriculumBuilder some more and realized this won't always be accurate. For example, `/csf-18/coursef/1/` is an unplugged lesson, but doesn't indicate as such in the url.